src/extra/normalize.css

```diff
@@ -135,6 +135,10 @@
   @media (--OSdark) {
     background-color: hsl(210deg 11% 10%);
   }
+
+  @nest [data-theme=dark] {
```

Comment:
```suggestion
  @nest [data-theme="dark"] & {
```

Reply:
🤦 I even got halfway there! 😆 First time using the proposed nest syntax. I don't hate it, but I do hate the inconsistency between it and the `@media` syntax.